### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ondrabus**
Updates the readme according to new Kontent brand guidelines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,14 @@ Welcome to the world of Kentico open source software. This is the home repositor
 
 ## Our products and related open-source projects
 
-### Kentico Kontent
-![Kentico Kontent logo](/images/kk-logo-hor-color-pos-sz-rgb.png)
+### Kontent by Kentico
+![Kontent by Kentico logo](/images/kk-logo-hor-color-pos-sz-rgb.png)
```
Comment (Simply007) [inline README.md:10]: Logo is not updated

Xperience logo might me changes as well

Comment (Simply007) [reply README.md]: fixed

**Comment by Simply007** [comment]
Also changed xperience.

Please take a look.

**Review by Simply007 — DISMISSED** [review]

**Comment by ondrabus** [comment]
I adjusted also MVPs and our job titles :-)

**Review by Simply007 — APPROVED** [review]

**Pull request merged**